spec/controllers/post_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :request do
+  it 'Expect a response code 200 and render correctly from the view index' do
+    get '/users/1/posts'
+    expect(response).to render_template(:index)
+    expect(response).to have_http_status(200)
+    expect(response.body).to include('User Number One')
```

Comment:
- [ ] [REQUIRED] Kindly take a look at some of the errors that we have, 8 tests are failing at the moment and most of them are tests pertaining to the body. Please take a look at this image.
- Please also note that the requirements state that you place the `users_spec.rb` and `posts_spec.rb` in a request directory which should be placed in the spec directory. 
``` 
spec/requests/ directory
```
<img width="672" alt="Screenshot 2023-01-25 at 22 33 36" src="https://user-images.githubusercontent.com/58692590/214696888-476de52e-8240-4617-9eb1-0680626a6f97.png">

Reply:
- Great job implementing the recommended changes @rbreva 👍🏽